aiida_quantumespresso/cli/data/structure.py

```diff
@@ -0,0 +1,34 @@
+# -*- coding: utf-8 -*-
+"""Command line utilities to create and inspect `StructureData` nodes."""
+from __future__ import absolute_import
+
+import click
+
+from aiida.cmdline.params import options
+from aiida.cmdline.utils import decorators, echo
+
+from . import cmd_data
+
+
+@cmd_data.group('structure')
+def cmd_structure():
+    """Commands to create and inspect `StructureData` nodes."""
+
+
+@cmd_structure.command('import')
```

Comment:
Just a general question, because I have never added click commands through a plugin before, but does this add the command `verdi data structure import` or does it add another possibility to that (already existing) `verdi` command?

Reply:
No, this is a CLI that I designed especially for the `aiida-quantumespresso` plugin. It installs the `aiida-quantumespresso` binary that works exactly like `verdi` and also comes with autocompletion. I did it this way because extending `verdi` through entry points only really supports adding new commands for `verdi data`, but I wanted my CLI to be able to do much more, such as launching calculations and work chains.

Reply:
Right - so for this particular thing, it could actually be added to `verdi`, but this is adding it to the `aiida-quantumespresso` CLI - very nice.
Now, however, I'm wondering whether it makes sense to do both, i.e. add it to the plugin CLI, but also to `verdi`?

Reply:
I think it does not make sense to add it also to `verdi`. If it would be possible to add it to the existing `verdi data structure`, maybe. But it has to be a new subcommand, e.g. `verdi data qe-structure`, which I think is ugly and since there is already `aiida-quantumespresso data structure`, it is best to not add this redundancy

Reply:
Right, I thought one could add to all sub-commands of `verdi data`. Yeah, then it doesn't make sense.